components/NavOverlay.tsx

```diff
@@ -73,6 +74,10 @@ export default function NavOverlay(props: {
               </Fragment>
             ))}
         <Divider />
+        <Box>
+          <ScheduleModal mobile={true} />
+        </Box>
```

Comment:
Nitpick: do you think the nav overlay should close after it gets selected?

Reply:
I agree! Addressed in https://github.com/cusec/2021/pull/99/commits/220924750909ee187f16367db4860edc8c1a516b